pkg/controller/oneagent/oneagent_controller.go

```diff
@@ -194,6 +198,45 @@ func (r *ReconcileOneAgent) Reconcile(request reconcile.Request) (reconcile.Resu
 	return reconcile.Result{RequeueAfter: 30 * time.Minute}, nil
 }
 
+func (r *ReconcileOneAgent) validateSecret(instance *dynatracev1alpha1.OneAgent) error {
+	secret := &corev1.Secret{}
+	err := r.client.Get(context.TODO(), client.ObjectKey{Namespace: instance.Namespace, Name: instance.Spec.Tokens}, secret)
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+
+	if errors.IsNotFound(err) {
+		var messageApiToken = fmt.Sprintf("The secret %s has not been found on namespace %s", instance.Spec.Tokens, instance.Namespace)
+		var messagePaasToken = fmt.Sprintf("The secret %s has not been found on namespace %s", instance.Spec.Tokens, instance.Namespace)
```

Comment:
It's the same message if you want to avoid duplication.

Reply:
Removed message once and reused the other